spec.html

```diff
@@ -51229,6 +51284,8 @@ <h1>Regular Expressions</h1>
 <emu-annex id="sec-additional-ecmascript-features-for-web-browsers" namespace="annexB" normative>
   <h1>Additional ECMAScript Features for Web Browsers</h1>
   <p>The ECMAScript language syntax and semantics defined in this annex are required when the ECMAScript host is a web browser. The content of this annex is normative but optional if the ECMAScript host is not a web browser.</p>
+  <p>Each of the features defined in this annex is specified as a set of modifications to the syntax, semantics, and/or built-in properties specified in the main body of this document. In some cases, those modifications appear in this annex, and in others, they appear (specially marked) in the main body.</p>
+  <p>When a modification to semantics appears in the main body (in an algorithm or early error rule), it is guarded by the condition that “<dfn variants="otherwise supports">the host supports</dfn>” the relevant feature. Web browsers are required to support all such features.</p>
```

Comment:
"Modification" reads off to me for the inline versions since it's just a branch. You use the phrase "special semantics" below. Thoughts on maybe here as well?

Reply:
The first of the new paras uses "modifications" in a more general sense than just "special semantics". I can sympathize that it doesn't feel like a modification if the feature is no longer monkey-patching the main body, but it's still a modification in the sense that the spec is different from how it would be without the feature.

I haven't found a better way to write the first para without using a word like "modifications".

Reply:
>  but it's still a modification in the sense that the spec is different from how it would be without the feature.

That reading of "modification" holds for any feature at any granularity in the spec, no?

> I haven't found a better way to write the first para without using a word like "modifications".

"Some features defined in this annex are specified in this annex, and some are specified in the main body of this document.

When the feature is specified in the main body, they are specially marked. In particular, specially marked algorithm steps and early error rules are guarded by the condition that “<dfn variants="otherwise supports">the host supports</dfn>” the relevant feature. Web browsers are required to support all such features."

Reply:
Okay, I've added a fixup commit along those lines. I used somewhat different wording for the second paragraph, but I think it's still consistent with what you wanted.

Reply:
Thanks!